index.html

```diff
@@ -163,74 +164,95 @@ <h2 class="title">Multi-Post Stories</h2>
-          <ol>
-          <li><a href="https://twitter.com/ahmadversee"><img src="./assets/Vector.svg" alt="twitter"></a></li>
-          <li><a href="https://www.linkedin.com/in/ahmad-sarfraz-ba90221a3/"><img src="./assets/Linkedin icon.svg" alt="Linkedin"></a></li>
-          <li><a href="#"><img src="./assets/Vector-2.svg" alt="m-icon"></a></li>
-          <li><a href="https://github.com/ahmadsarfrazx"><img src="./assets/Vector-1.svg" alt="github"></a></li>
-          <li><a href="#"><img src="./assets/Vector-3.svg" alt="victory"></a></li>
-          </ol>
-        </div>
-      <div class="action">
-        <button type="button" class="default_button">Get my resume</button>
+        feature or website Look through some of my work and experience! If
+        you like what you see and have a project you need coded, don’t
+        hesitate to contact me.
+       </p>
+      
+        <a class="contact-title-2" href="#">LET’S CONNECT</a>
+        <ul class="social-media-2">
+          <li>
+            <a href="#"
+              ><img src="assets/tw.png" alt="Twitter"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img
+                src="assets/linkedin.png"
+                alt="LinkedIn"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="assets/medium.png" alt="Medium"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="assets/github.png" alt="Github"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="assets/v.png" alt="V"
+            /></a>
+          </li>
+        </ul>
+        <button class="project-button" type="button">Get my resume</button>
       </div>
-    </div>
-    <div class="about-right-section-">
-      <ol class="about_ul">
-        <h3 class="about_sub_title">
-          Languages
-          <img
-            class="down_arrow_icon"id="icons"
-            src="./assets/down-arrow-icon.png"
-            alt="down"
-          />
-        </h3>
-        <li class="ul_item">
-          <img src="./assets/js-icon.png" alt="Javascript" /><span
-            >Javascript</span
-          >
+      <div class="right-box-of-about-myself">
+       <ul class="skill-list">
+        <li class="my-skill">
+          <div class="skill-container">
+            <h3 class="skill-title">Languages</h3>
+            <button class="btn-chev" type="button">
+              <img src="assets/chev-dow.svg" alt="" />
+            </button>
+          </div>
+          <ul class="language-list">
+            <li>
+              <img src="assets/javascript.svg" alt="JavaScript" />
+              JavaScript
+            </li>
+            <li>
+              <img src="assets/html.svg" alt="HTML" />
+              HTML
+            </li>
+            <li>
+              <img src="assets/css.svg" alt="CSS" />
+              CSS
+            </li>
```

Comment:
- [ ] While you were implementing the changes, you forgot to remove the list style(dots beside the languages, framework and skills) also, Javascript, html and CSS are in rows, not column.
### Your webpage
![image](https://user-images.githubusercontent.com/86200356/200123393-a0d1a7b2-072b-4b43-bcf1-274c27cf8107.png)
### Figma
![image](https://user-images.githubusercontent.com/86200356/200123451-6bed23aa-e26c-4686-a350-e153becfa018.png)